src/mono/wasm/debugger/tests/debugger-test/debugger-test.cs

```diff
@@ -849,3 +849,16 @@ public static void RunDebuggerBreak()
         VisibleMethodDebuggerBreak();
     }
 }
+
+public class DebugTypeFull
+{
+    public static void CallToEvaluateLocal(string asm_base64, string pdb_base64)
+    {
+        var asm = System.Reflection.Assembly.LoadFrom("debugger-test-with-full-debug-type.dll");
+        var myType = asm.GetType("DebuggerTests.ClassToInspectWithDebugTypeFull");
+        var myMethod = myType.GetConstructor(new Type[] { });
+        var a = myMethod.Invoke(new object[]{});
+        Console.WriteLine(a);
```

Comment:
Should we leave this writing to the console here?

Reply:
Removed :)